tools/onnx/sarif/gen_sarif.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+ROOT=$(pwd)/../../..
+SARIF_DIR=torch/onnx/_internal/diagnostics/infra/sarif_om
+
+# SARIF version
+SARIF_VERSION=2.1.0
+SARIF_SCHEMA_LINK=https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/schemas/sarif-schema-2.1.0.json
```

Comment:
Would this json/version change very often? Curious about the maintenance work on this.

Reply:
No, I don't expect it to change. However soon we should add CI checks to ensure any modification to these files are made through code generation.